tools/cockpit.spec

```diff
@@ -577,15 +580,18 @@ getent passwd cockpit-ws >/dev/null || useradd -r -g cockpit-ws -d / -s /sbin/no
 
 %post ws
 %systemd_post cockpit.socket
+%systemd_post cockpit-issue.service
```

Comment:
Is that enough to actually enable the service? Also, should we actually already enable this, given that issue.d/ is not yet supported in Fedora/CentOS/RHEL (or anywhere else)?